From e8c812380f5bb0dd1d6a89754e8157fc1ad392d4 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Wed, 18 Dec 2019 17:21:31 +0800
Subject: [PATCH] thermal: rk_virtual_thermal: select POWER_SUPPLY when
 RK_VIRTUAL_THERMAL enabled

Fix build error without POWER_SUPPLY.

Change-Id: I259f914e6e9a945ffdd8676639916a7e07f92d40
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 drivers/thermal/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/thermal/Kconfig b/drivers/thermal/Kconfig
index 07d11279f3ab..703ab32a4d29 100644
--- a/drivers/thermal/Kconfig
+++ b/drivers/thermal/Kconfig
@@ -265,6 +265,7 @@ config ROCKCHIP_THERMAL
 config RK_VIRTUAL_THERMAL
 	tristate "rk_virtual thermal driver"
 	depends on ROCKCHIP_THERMAL
+	select POWER_SUPPLY
 	help
 	  Rk virtual thermal driver provides virtual temperature support for
 	  SOCs without tsadc module. It supports one critical trip point.
-- 
2.35.3

